pkg/scheduler/framework/v1alpha1/framework.go

```diff
@@ -287,13 +303,12 @@ func (f *framework) RunPrefilterPlugins(
 // Meanwhile, the failure message and status are set for the given node.
 func (f *framework) RunFilterPlugins(pc *PluginContext,
 	pod *v1.Pod, nodeName string) *Status {
-
-	for _, p := range f.filterPlugins {
-		status := p.Filter(pc, pod, nodeName)
+	for _, pl := range f.filterPlugins {
+		status := pl.Filter(pc, pod, nodeName)
 		if !status.IsSuccess() {
 			if status.Code() != Unschedulable {
-				errMsg := fmt.Sprintf("RunFilterPlugins: error while running %s filter plugin for pod %s: %s",
-					p.Name(), pod.Name, status.Message())
+				errMsg := fmt.Sprintf("error while running %v filter plugin for pod %v: %v",
```

Comment:
nit: I would use %q for the plugin and pod names so that they get logged quoted.

Reply:
> nit: I would use %q for the plugin and pod names so that they get logged quoted.

I'm trying to be consistent with other logs here, and I'll open a follow-up PR for this.

Reply:
> They could, I think. lets leave that for a separate PR.

Removed from this